EMIP/1-1000/EMIP00764.xml

```diff
@@ -57,6 +57,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         </extent>
                      </supportDesc>
                   </objectDesc>
+                  <additions>
+                     <list>
+                        <item xml:id="a1">
+                           <locus target="2r"/>
+                           <desc type="PoemQene"><ref type="work" corresp="LIT7324QeneIfatara">Śāhlǝka-qǝne</ref> on the upper margin.</desc>
+                           <q xml:lang="gez">
+                              ሣህልከ፡
+                              <l n="1">ኢፈጠረ፡ ስመ፡ በላዕለ፡ ስሙ፡</l> 
+                              <l n="2">ወል<sic>ዳ</sic>፡ ሥላሴ፡ መስፍን፡ ዘኢይትሜካህ፡ በአቅ<supplied reason="subaudible" resp="CH">ሙ</supplied>፡</l>
```

Comment:
As I understand the terminology, the ሙ is not "subaudible" but rather "lost" (because the manuscript is materially torn here)